redhawk/common/xpath.py

```diff
@@ -130,6 +132,8 @@
 import re
 import sys
 
+from six.moves import filter
```

Comment:
This is the only file that the six module is now being imported for. In this file, filter is used only at Line 226. Replacing it with list(filter(..)) will enable us to remove this import, thereby removing six as a dependency altogether.